src/test/java/org/example/TaskServiceTest.java

```diff
@@ -0,0 +1,112 @@
+package org.example;
+
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class TaskServiceTest {
+    TaskService taskService;
+    ArrayList<Task> taskList = new ArrayList<>();
+
+    @Test
+    public void shouldAddTaskToTheList() {
+        Task task1 = new Task("Zadanie 1", false);
+        Task task2 = new Task("Zadanie 2", true);
+        Task task3 = new Task("Zadanie 3", false);
+        String newTask = "Tidy up your room";
+
+        Given:
+        taskList.add(task1);
+        taskList.add(task2);
+        taskList.add(task3);
+        this.taskService = new TaskService(taskList);
+        int sizeBeforeRemoving = taskService.getAll().size();
+        When:
+        taskService.add(newTask);
+
+        Then:
+        assertThat(taskService.getAll().size()).isEqualTo(sizeBeforeRemoving + 1);
+    }
+
+    @Test
+    public void shouldRemoveElementFromList() {
+        Task task1 = new Task("Zadanie 1", false);
+        Task task2 = new Task("Zadanie 2", true);
+        Task task3 = new Task("Zadanie 3", false);
+        int indexToRemove = 1;
+
+        Given:
+        taskList.add(task1);
+        taskList.add(task2);
+        taskList.add(task3);
+        this.taskService = new TaskService(taskList);
```

Comment:
masz pole w klasie które za każdym razem podmieniasz, to może być po prostu pole w ramach testów, chciałbym żeby to wyglądało tak: `TaskService taskService = new TaskService(List.of(...));` wtedy każdy test ma czyste konto i nie musisz znowu scrolować do góry, takie wspólne pola można używać, ale to z reguły są jakieś utilsowe klasy, albo stałe i nie zmieniają się nigdy